js/src/Glacier2/.gitignore

```diff
@@ -4,8 +4,6 @@ PermissionsVerifier.js
 PermissionsVerifier.d.ts
 Router.js
 Router.d.ts
-RouterF.js
-RouterF.d.ts
```

Comment:
There's still some references to 'PermissionsVerifierF' in this file that should be removed:
-  PermissionsVerifierF.js
-  PermissionsVerifierF.d.ts